docker/images/outrigger/README.md

```diff
@@ -0,0 +1,56 @@
+# Outrigger Docker Image
+
+# Building
+
+To build simply run:
+
+```
+docker build -t outrigger .
+``` 
+
+in the directory this README is currently in.
+
+# Running
+
+To run the outrigger server simply run:
+
+```
+docker run -d -p 8080:8080 -v `pwd`/juttles:/opt/juttles --name outrigger outrigger
```

Comment:
I think there's a way to do this by documenting to share the right external path to an internal path such as `/opt/outrigger` so the working directory is always set to that and we'd load any `juttle-config.js` or `juttle-config.json` present there. Let me try this out locally and document accordingly.